cloud/data_source_cloud_environment.go

```diff
@@ -40,16 +40,16 @@ func dataSourceCloudEnvironment() *schema.Resource {
 			},
 		},
 		Schema: map[string]*schema.Schema{
-			"organization": {
+			"id": {
```

Comment:
I think we can keep use the `name` argument here when using data source

Reply:
Sure, I can switch that back

Reply:
switched back to use name